tools/reviewer/web/src/app/shared/services/firebase.service.ts

```diff
@@ -45,6 +45,10 @@ export class FirebaseService {
   }
 
   updateDiff(diff: Diff): Observable<void> {
+    // Milliseconds since Jan 01 1970. (UTC)
```

Comment:
This is commonly called `Unix timestamp`.

Reply:
Yeah, I know. I wanted to emphasize that it's unix timestamp in milliseconds.

Reply:
So maybe rephease to `Unix timestamp in milliseconds`?

Reply:
Ok